This test is failing when run under pytest 8 so we simply remove it.
https://github.com/pylint-dev/pylint/issues/9477
https://github.com/pylint-dev/pylint/pull/9576

--- pylint-3.2.7/tests/test_regr.py.orig
+++ pylint-3.2.7/tests/test_regr.py
@@ -56,7 +56,6 @@
     "file_names, check",
     [
         (["package.__init__"], Equals("")),
-        (["precedence_test"], Equals("")),
         (["import_package_subpackage_module"], Equals("")),
         (["pylint.checkers.__init__"], lambda x: "__path__" not in x),
         ([join(REGR_DATA, "classdoc_usage.py")], Equals("")),
